kitsune/questions/views.py

```diff
@@ -410,7 +410,7 @@ def question_details(
 
     extra_kwargs.update(ans_)
 
-    products = Product.objects.filter(visible=True)
+    products = Product.objects.with_question_forums(request)
```

Comment:
Won't this change the current behavior?

Reply:
Yes, this is needed to fix https://github.com/mozilla/sumo/issues/1008: we don't want to add products without a support forum to the edit product dropdown

Reply:
My comment was related to visible. Don't we need to filter on both the `with_question_forums` and `visible=True`?

Reply:
Since the end-user-facing logic which dictates whether a question can be posted to a product forum doesn't take into account that product's `visible` state (instead requiring its existence in the question `config.py` file, and an appropriate QuestionLocale row), I don't think so, no. Otherwise we could get into the state where end-users can still post into a product forum, but contributors can't move questions into it, nor see it listed on the support forums page.